lib/ui/page/home/page/chat/info/view.dart

```diff
@@ -20,7 +20,7 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:get/get.dart';
 import 'package:scrollable_positioned_list/scrollable_positioned_list.dart';
-
+import '/ui/widget/line_divider.dart';
```

Comment:
I suggest not removing the space between package and relative/absolute imports just like in all the files of source code.